src/Collapsible/index.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-did-update-set-state */
```

Comment:
I'm not a fan of disabling this. It should be relatively easy to refactor (see https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-did-update-set-state.md).

Reply:
Isn't it ok in this case since it's wrapped in a condition checking the previous props? https://reactjs.org/docs/react-component.html#componentdidupdate
```jsx
componentDidUpdate(prevProps) {
  // Typical usage (don't forget to compare props):
  if (this.props.userID !== prevProps.userID) {
    this.fetchData(this.props.userID);
  }
}
```
"You may call setState() immediately in componentDidUpdate() but note that it must be wrapped in a condition like in the example above"

Reply:
Added comments explaining why it's disabled